App crashes after boot

Happens (on Android 9) since a while and also on latest master. 
```
--------- beginning of crash
01-17 18:20:47.675  3711  3711 E AndroidRuntime: FATAL EXCEPTION: main
01-17 18:20:47.675  3711  3711 E AndroidRuntime: Process: net.programmierecke.radiodroid2, PID: 3711
01-17 18:20:47.675  3711  3711 E AndroidRuntime: java.lang.RuntimeException: Unable to instantiate service net.programmierecke.radiodroid2.service.RadioDroidBrowserService: java.lang.ClassNotFoundException: Didn't find class "net.programmierecke.radiodroid2.service.RadioDroidBrowserService" on path: DexPathList[[zip file "/data/app/net.programmierecke.radiodroid2-cuIH_zn1wnUIiu1_sFhG9w==/base.apk"],nativeLibraryDirectories=[/data/app/net.programmierecke.radiodroid2-cuIH_zn1wnUIiu1_sFhG9w==/lib/arm64, /system/lib64, /vendor/lib64]]
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.app.ActivityThread.handleCreateService(ActivityThread.java:3557)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.app.ActivityThread.access$1300(ActivityThread.java:200)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1672)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.os.Looper.loop(Looper.java:193)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6718)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:491)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
01-17 18:20:47.675  3711  3711 E AndroidRuntime: Caused by: java.lang.ClassNotFoundException: Didn't find class "net.programmierecke.radiodroid2.service.RadioDroidBrowserService" on path: DexPathList[[zip file "/data/app/net.programmierecke.radiodroid2-cuIH_zn1wnUIiu1_sFhG9w==/base.apk"],nativeLibraryDirectories=[/data/app/net.programmierecke.radiodroid2-cuIH_zn1wnUIiu1_sFhG9w==/lib/arm64, /system/lib64, /vendor/lib64]]
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at dalvik.system.BaseDexClassLoader.findClass(BaseDexClassLoader.java:134)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at java.lang.ClassLoader.loadClass(ClassLoader.java:379)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at java.lang.ClassLoader.loadClass(ClassLoader.java:312)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.app.AppComponentFactory.instantiateService(AppComponentFactory.java:103)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at androidx.core.app.CoreComponentFactory.instantiateService(CoreComponentFactory.java:66)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        at android.app.ActivityThread.handleCreateService(ActivityThread.java:3552)
01-17 18:20:47.675  3711  3711 E AndroidRuntime:        ... 8 more

--- a/app/src/main/AndroidManifest.xml
+++ b/app/src/main/AndroidManifest.xml
@@ -51,7 +51,7 @@
         </service>
 
         <service
-            android:name="net.programmierecke.radiodroid2.service.RadioDroidBrowserService"
+            android:name="net.programmierecke.radiodroid2.RadioDroidBrowserService"
             android:exported="true"
             tools:ignore="ExportedService">
             <intent-filter>

